Robinhood/Robinhood.py

```diff
@@ -73,11 +56,75 @@ class Robinhood:
     logger = logging.getLogger('Robinhood')
     logger.addHandler(logging.NullHandler())
 
+    from . import exceptions
+
+    class Bounds(Enum):
```

Comment:
really don't like enums being moved inside class.  They aren't as accessible to tests

Reply:
Agreed they should be Robinhood.exceptions, not Robinhood.Robinhood.exceptions .. but see above re whole thing is a Class today.